src/pipecat/processors/aggregators/openai_llm_context.py

```diff
@@ -196,25 +196,30 @@ async def call_function(
         # Push a SystemFrame downstream. This frame will let our assistant context aggregator
         # know that we are in the middle of a function call. Some contexts/aggregators may
         # not need this. But some definitely do (Anthropic, for example).
-        await llm.push_frame(
-            FunctionCallInProgressFrame(
+        # Also push a SystemFrame upstream for use by other processors, like STTMuteFilter.
+        progress_frame = FunctionCallInProgressFrame(
+            function_name=function_name,
+            tool_call_id=tool_call_id,
+            arguments=arguments,
+        )
+
+        # Push frame both downstream and upstream
+        await llm.push_frame(progress_frame, FrameDirection.DOWNSTREAM)
+        await llm.push_frame(progress_frame, FrameDirection.UPSTREAM)
```

Comment:
Oh, I put another comment about this before. We should not reuse the same frame.

Reply:
Is that for debugging purposes?

Reply:
I saw your comment in Slack. You want unique instances pushed in the frames. 👍